### PR TITLE
Mult test en

### PR DESCRIPTION
**Pull request by adlv888**

### CODE REVIEW

File: apps/scripts/AliveTest.c [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+void XORI(int a){
+    D_MEM_SHARED[4] = a^42;
+}
+void SLLI(int a){
+    a=a<<3;
+    if(a==64)
+    	D_MEM_SHARED[5] = 1;
+}
+void SRLI(int a){
+    __asm__("lui a5,0x41;"
```
Comment (DL8) [inline apps/scripts/AliveTest.c:37]: you can replace the "lui-addi" sequence with a simple "li". this is a pseudoinstruction that will be automagically be translated into "lui-addi" sequence if needed (or just "addi" if the constant is small enough)

File: apps/scripts/AliveTest.c [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+void XORI(int a){
+    D_MEM_SHARED[4] = a^42;
+}
+void SLLI(int a){
+    a=a<<3;
+    if(a==64)
+    	D_MEM_SHARED[5] = 1;
+}
+void SRLI(int a){
```
Comment (DL8) [inline apps/scripts/AliveTest.c:36]: if you want to force a logical shift without writing complicated assembly, you can declare `a` as unsigned int

File: apps/scripts/AliveTest.c [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+
+void SLT(int a, int b){
+	D_MEM_SHARED[9]= a<b ? 1 : 0;
+} 
+
+/////////////////////////////////////
+void SLTU(int a){
+	D_MEM_SHARED[10]= a==0 ? 0 : 1;
```
Comment (DL8) [inline apps/scripts/AliveTest.c:64]: this expression may translate into stlu, but it's not that straightforward. perhaps you can use a signature like slt, but with unsigned ints:
```c
void SLTU(unsigned int a, unsigned int b)
```

**Review by amichai-bd — DISMISSED** [review]
Good job!

**Review by amichai-bd — APPROVED** [review]
looking good

**Pull request merged**